packages/search-ui-app-search-connector/src/__tests__/responseAdapter.test.js

```diff
@@ -189,10 +251,113 @@ const adaptedResponse = {
         ]
       }
     ]
-  }
+  },
 };
 
-const adaptedEmptyResponse = { requestId: "1234", results: [] };
+const adaptedEmptyResponse = {
+  rawResponse: {
+    info: {
+      meta: {
+        request_id: "1234",
+      },
+    },
+    rawResults: [],
+  },
+  requestId: "1234",
+  results: []
+};
+
+const adaptedResponseWithEmptyFacetValue = {
```

Comment:
I merged master into this PR from the Github UI and it looks like we now have duplicate declarations for `adaptedResponseWithEmptyFacetValue`. Could you please resolve this manually? Sorry about that.